.stylelintrc

```diff
@@ -138,6 +138,9 @@
   "selector-max-compound-selectors": 3,
   "selector-max-empty-lines": 0,
   "selector-max-specificity": "0,4,1",
+  "selector-nested-pattern": ["(?:&(?=[\\s:.,\\[])|[^&])*", {
```

Comment:
Just for people who have trouble reading this monstrosity, it says: all characters in the selector must either be not an ampersand (`[^&]`), or they must be an ampersand followed by one of `\s`, `:`, `.`, `,`, `[`

Reply:
I'm assuming the `\s` is for matching a single [white-space](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#special-white-space), correct?

Reply:
Yep

Reply:
Hey, wait. I'm a little confused by the escaping here. Is this not a typical JS regex format? For instance, wouldn't you also need to escape `.` and `]`? Because wouldn't the `.` be interpreted as any character is allowed?

Reply:
Good question. I _think_ the dot doesn't need it because it's inside the square brackets, and the square bracket _is_ escaped. I have to double escape them because in stylelintrc's config for some reason.